src/main/python/pybuilder/utils.py

```diff
@@ -304,3 +305,269 @@ def instrumented_target(*args, **kwargs):
             msg += "\n" + chained_message
         ex = Exception(msg)
         raise_exception(ex, result.args[1])
+
+
+if sys.version_info[0] == 2 and sys.version_info[1] == 6:  # if Python is 2.6
+    # Backport of OrderedDict() class that runs on Python 2.4, 2.5, 2.6, 2.7 and pypy.
+    # Passes Python2.7's test suite and incorporates all the latest updates.
+
+    try:
+        from thread import get_ident as _get_ident
+    except ImportError:
+        from dummy_thread import get_ident as _get_ident
+
+    try:
+        from _abcoll import KeysView, ValuesView, ItemsView
+    except ImportError:
+        pass
+
+    class OrderedDict(dict):
```

Comment:
I think it's great that we have this, there are plenty of other places where I wanted to use an OrderedDict. Some of these are in the core startup code that runs before installing dependencies, which leads to a chicken/egg problem.. now we can always use an ordered dict.